src/bindgen/ir/constant.rs

```diff
@@ -104,6 +110,9 @@ impl Literal {
                 right.rename_for_config(config);
             }
             Literal::Expr(_) => {}
+            Literal::Cast { ref mut value, .. } => {
```

Comment:
The type should also be renamed to account for the configuration. This should be trivial to test.